src/style.css

```diff
@@ -1,8 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;600&display=swap');
+
+/* width */
+::-webkit-scrollbar {
+  width: 10px;
+  border-radius: 10px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  border-radius: 10px;
+  background: #888;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  border-radius: 10px;
+  background: #f1f1f1;
+}
+
+:root {
+  --font1: 'Poppins', sans-serif;
+  --bg-color: #252525;
+  --outer-shadow: -5px -5px 10px rgba(255, 255, 255, 0.079), 5px 5px 10px rgba(0, 0, 0, 0.21);
+  --inner-shadow: -5px -5px 10px rgba(255, 255, 255, 0.079), 5px 5px 10px rgba(0, 0, 0, 0.21);
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
+
 * {
   margin: 0;
   padding: 0;
 }
 
-.hello {
+body {
+  width: 100%;
+  height: calc(100vh - 60px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: var(--font1);
+  background-color: var(--bg-color);
+  color: white;
+  padding-top: 60px;
+}
+
+header {
+  font-size: 40px;
+}
+
+form {
+  width: 90%;
+  max-width: 800px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+  border-radius: 7px;
+  box-shadow: var(--outer-shadow);
+  margin: 30px 0;
+}
+
+form input {
+  width: 300px;
+  background-color: var(--bg-color);
+  border: none;
+  outline: none;
+  color: white;
+  padding: 10px;
+  border-radius: 7px;
+  box-shadow: var(--outer-shadow), inset 0 0 15px rgba(0, 0, 0, 0.371);
+  font-size: 16px;
+}
+
+input::placeholder {
+  color: rgba(255, 255, 255, 0.656);
+}
+
+button {
+  padding: 10px 20px;
+  background-color: rgba(255, 255, 255, 0);
+  outline: none;
+  color: rgb(118, 175, 255);
+  border: 2px solid rgb(11, 112, 255);
+  border-radius: 7px;
+  font-weight: 600;
+  font-size: 14px;
+}
+
+ul {
```

Comment:
Good job so far, the style of your app looks nice but I will highly recommend you follow the design given in the video.

- [ ] For this project, the to-do list app, you're required to style the app to look as close as the one in the video. Kindly modify your styles for the entire app to match the design given in the project requirement 👍🏽

**The Required Design**
<img width="1092" alt="Screenshot 2023-02-22 at 12 18 24 AM" src="https://user-images.githubusercontent.com/50941074/220489059-b7180571-df8a-49a7-8104-8b652bac5844.png">

**Your Design**
<img width="980" alt="Screenshot 2023-02-22 at 12 19 25 AM" src="https://user-images.githubusercontent.com/50941074/220488999-f5e8ac65-27f7-4980-86fe-89a3e6c6b3c7.png">

Reply:
Sure I will, But isn't that design boring? I thought a custom design would be appreciated!
But no worries, I will do that design